### PR TITLE
Indicate inactive dyn notch on looprates < 2kHz

### PR DESCRIPTION
**Pull request by KarateBrot**
While testing some PR's I noticed that the configurator never indicated in any way that the dynamic notch is disabled on PID looprates smaller than 2kHz. So I added some indication for that.

### Behavior if PID looprate < 2kHz:

**BF 4.2 and earlier:** 
- dynamic notch GUI gets hidden
- Added note about disability at looprates < 2kHz

**BF 4.3:**
- dynamic notch switch is off, indicating disability

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
This is useful and worth including in 10.8 with 4.3 because some people will be running 2k PID loops.

**Comment by KarateBrot** [comment]
> This is useful and worth including in 10.8 with 4.3 because some people will be running 2k PID loops.

Fortunately 2k is still enough to run the dynamic notch. But it's on the edge.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by Asizon** [comment]
I dont think any user uses 2k pidloop from 2017-2018 year. All F4 boards runs 2k perfectly.

**Review by Asizon — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
